### ISSUE TITLE
Gui hulp voor ketzer aanmaken

### ISSUE THREAD

**Issue by brussee** [csrdelft/csrdelft.nl#652]
Uitbreiding van huidige popup met keuzes: [stappenplan van vragen](https://nl.wikipedia.org/wiki/Expertsysteem) die je leidt tot de ketzer die de gebruiker wil maken.
Eventueel ketzers weer op een hoop gooien (muv rechten-groepen) en feature flags gebruiken, of velden die niet zijn ingevult = uit?

**Comment by mathcals** [comment]
Dat is ongeveer het idee achter de ketzertovenaar branch volgens mij.

**Comment by jptned** [comment]
Klopt. Daar ben ik inderdaad al mee bezig, al was daar nog geen issue voor. Dit is wat ik aan het uitwerken ben:

![Artboard 1-100](https://user-images.githubusercontent.com/8656358/77834875-68ea7000-7148-11ea-9f63-c5aa76f24d6d.jpg)

**Comment by brussee** [comment]
Mooi! Ik zie nog niet de kosten terugkomen (voor deelname of aanschafprijs van iets).

Misschien kun je de categorie-vraag impliciet maken door te vragen naar wie de organisatie is (commissie of huis bijv.) en evt. doelgroep?
